src/components/PullRequestList/index.tsx

```diff
@@ -6,6 +6,14 @@ import CardShimmer from '@/components/Loaders/cardShimmer';
 import useFetch from '@/hooks/useFetch';
 import styles from './PullRequestList.module.scss';
 
+const SCREEN_HEIGHT = globalThis.innerHeight;
+const SCREEN_WIDTH = globalThis.innerWidth;
+const CARD_HEIGHT = 147;
+const NUMBER_OF_CARDS_FOR_MOBILE_DISPLAYS = 1;
+const NUMBER_OF_CARDS_FOR_DESKTOP = 3;
+const SMALL_SCREEN_WIDTH_THRESHOLD = 700;
+const HORIZONTAL_NUMBER_OF_CARDS = (SCREEN_WIDTH < SMALL_SCREEN_WIDTH_THRESHOLD
+  ? NUMBER_OF_CARDS_FOR_MOBILE_DISPLAYS : NUMBER_OF_CARDS_FOR_DESKTOP);
```

Comment:
This approach works great for resolutions above 720p and for phone screens.
But we have 2 horizontal cards for a small desktop screen with a resolution of 800 x 600pixels.
In this case, we end up loading more cards than required.

Can we have some generic approach for any screen width and screen height? Can we calculate the number of cards in a row?

Reply:
Yes, right. Could you please suggest any approach or anything that can help me with this

Reply:
I have found a solution to this problem. I will update the PR and then you can check it